spring-integration-gcp/src/main/java/org/springframework/integration/gcp/pubsub/outbound/PubSubMessageHandler.java

```diff
@@ -61,6 +62,8 @@
 
 	private Expression publishTimeoutExpression = new ValueExpression<>(DEFAULT_PUBLISH_TIMEOUT);
 
+	private ListenableFutureCallback<String> publishFutureCallback;
```

Comment:
Maybe just `publishCallback`?

Reply:
+1 and setter would sound better - `setPublishCallback`